src/views/Change.vue

```diff
@@ -55,6 +55,34 @@
           sectionNumber="3."
           :validate="getAppValidate"
         />
+        <template v-if="isRoleStaff">
+          <h2 class="mt-10">{{'4.'}} Court Order and Plan of Arrangement</h2>
+          <div class="py-4">
+            If this filing is pursuant to a court order, enter the court order number. If this
+            filing is pursuant to a plan of arrangement, enter the court order number and select
+            Plan of Arrangement.
+          </div>
+
+          <div :class="{'invalid-section': invalidCourtOrder}">
+            <CourtOrderPoaShared
+              id="court-order"
+              :autoValidation="getAppValidate"
+              :draftCourtOrderNumber="getFileNumber"
+              :hasDraftPlanOfArrangement="getHasPlanOfArrangement"
+              :invalidSection="invalidCourtOrder"
+              @emitCourtNumber="setFileNumber($event)"
+              @emitPoa="setHasPlanOfArrangement($event)"
+              @emitValid="setValidCourtOrder($event)"
+            />
+          </div>
```

Comment:
It would be cleaner to move this header + blurb + sub-component into a new component (like the other ones).